### ISSUE TITLE
Add `copyPasteEnabled` to the definition file

### ISSUE THREAD

**Issue by AMBudnik** [handsontable/handsontable#8850]
### Description

Add `copyPasteEnabled` to the definition file.
We [have it in the docs](https://handsontable.com/docs/column-hiding/#step-4-set-up-copy-and-paste-behavior) but it does not have a definition. 

### Links
1. Definiton of `hiddenColumns` namespace - https://github.com/handsontable/handsontable/blob/master/handsontable.d.ts#L2573
2. Docs for `hiddenColumns` - https://handsontable.com/docs/column-hiding/#step-4-set-up-copy-and-paste-behavior

Inform [forum](https://forum.handsontable.com/t/copypasteenabled-missing-from-hiddencolumns-setting/5576)

**Comment by NeSy** [comment]
Workaround :
The functionality seems to be working if we cast the type to what the Typescript compiler is expecting.

```
import { Settings as ForcedHiddenColumnsSettings } from 'handsontable/plugins/hiddenColumns';
...
      hiddenColumns: {
        copyPasteEnabled: false,
      } as ForcedHiddenColumnsSettings,
```

**Comment by adrianszymanski89** [comment]
Hi @NeSy 

Thank you for the workaround. It's a good solution until we fix it on our side.

**Comment by AMBudnik** [comment]
Hi @NeSy 

We just released Handsontable v14.0.0 when the missing definition is added (this PR https://github.com/handsontable/handsontable/pull/10492)

Here https://handsontable.com/docs/javascript-data-grid/migration-from-13.1-to-14.0/ you can check our migration guide. Then here https://handsontable.com/docs/javascript-data-grid/release-notes/#_14-0-0 is a full list of changes.

If you'd have any questions please feel free to message me here or at support@handsontable.com
